### PR TITLE
Configure prettier

### PR DESCRIPTION
**Pull request by mdubus**
## What's wrong ?
Prettier was installed but the rules weren't configured properly. There was different coding styles in different files. Lack of uniformity. 

## What's inside this PR ?
Edit of the configuration file, and formatting of every file regarding the same specification.
Please take a look at the rules to see if it seems coherent to you 🙏

### CODE REVIEW

**Comment by bidoubiwa** [comment]
bors try

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**/.cache/
+.github/
+**/dist/
```
Comment (bidoubiwa) [inline .prettierignore:3]:
```suggestion
**/dist/

```

**Review by bidoubiwa — CHANGES_REQUESTED** [review]
In the contributing.md there is still the following: 

```
# Linter with auto-correct
yarn lint-fix
# Prettier for Markdown (auto-correct)
yarn format:md
# Prettier for SCSS (auto-correct)
yarn format:scss
```

Could you change the `lint-fix` and also test the format to see if they work properly?

**Comment by bidoubiwa** [comment]
Also in the contributing.md apparently the linter recognizes that the code is `.js` thus he does not lint what is not `.js`. 

So you have the following at the bottom of the file: 

```
"version": "X.X.X",

export default 'X.X.X'
```

Could you replace the double quotes with simple quotes?

**Comment by mdubus** [comment]
> In the contributing.md there is still the following:
> 
> ```
> # Linter with auto-correct
> yarn lint-fix
> # Prettier for Markdown (auto-correct)
> yarn format:md
> # Prettier for SCSS (auto-correct)
> yarn format:scss
> ```
> 
> Could you change the `lint-fix` and also test the format to see if they work properly?

Nice catch ! I tested them and they work perfectly 👌

**Review by bidoubiwa — APPROVED** [review]
bors merge

**Pull request merged**